src/main/webapp/index.html

```diff
@@ -3,30 +3,38 @@
   <head>
     <meta charset="UTF-8">
     <title>Is Covid Near Me?</title>
-    <link rel="stylesheet" href="main.css">
+    <link rel="stylesheet" href="style.css">
+    <link rel="shortcut icon" type="image/png" href="img/favicon.png">
     <script src="script.js"></script>
   </head>
   <body>
     <div class="content">
       <h1>Is Covid near me?</h1>
+
       <form id="myForm" autocomplete="off">
-    <div class="user-box">
-      <input type="text" name="state" id="state" required="">
-      <label>State</label>
-    </div>
-    <div class="user-box">
-      <input type="text" name="county" id="county" required="">
-      <label>County</label>
-    </div>
-    <a href="#" onclick="getData();">
-      <span></span>
-      <span></span>
-      <span></span>
-      <span></span>
-      Submit 
-    </a>
-  </form><br>
-      <h2 id=results style="font-weight: normal"></h2>
+
+        <div class="user-box">
+            <input type="text" id="state" required="">
+            <label>State</label>
+        </div>
+
```

Comment:
Oh, and about any confusion, I didn't change anything here, I just added spaces/newlines and fixed tabbing a little just to make things clearer